assets/js/modules/adsense/components/dashboard/DashboardTopEarningPagesWidget.js

```diff
@@ -66,7 +66,9 @@ function DashboardTopEarningPagesWidget( { Widget, WidgetReportZero, WidgetRepor
 
 		return {
 			isAdSenseLinked: select( MODULES_ANALYTICS ).getAdsenseLinked(),
-			analyticsMainURL: select( MODULES_ANALYTICS ).getServiceURL(),
+			analyticsMainURL: select( MODULES_ANALYTICS ).getServiceReportURL( 'content-publisher-overview', {
+				'_u.date00': startDate, '_u.date01': endDate,
```

Comment:
We can't pass raw dates like this because they need to be formatted as `YYYYMMDD` as noted in the IB.
We added the `generateDateRangeArgs` utilities to handle this for us, as well as abstracting the relationship between arg name and specific dates which is not obvious just by looking at it.

Let's update it to use `generateDateRangeArgs( { startDate, endDate } )` instead 👍 
Use care to make sure we're sourcing the utility from Analytics utils though since AdSense has its own version by the same name, which is different.